### ISSUE TITLE
CRITICAL : Unable to provision Standard cluster with ibm_container_cluster instance

### ISSUE THREAD

**Issue by M-Rushabh** [IBM-Cloud/terraform-provider-ibm#515]
Hi there,

### Terraform Version
Terraform v0.11.7
IBM Cloud Provider version 0.14.1

### Affected Resource(s)
- ibm_container_cluster

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
resource "ibm_container_cluster" "testacc" {
  name = "clusterushabhh"
  datacenter = "dal12"
  machine_type = "u2c.2x4"
  hardware = "shared"
  region = "us-south"

private_vlan_id="2260213"
 }
```

### Debug Output
ibm_container_cluster.testacc: Still creating... (1h30m1s elapsed)

Error: Error applying plan:

1 error(s) occurred:

* ibm_container_cluster.testacc: 1 error(s) occurred:

* ibm_container_cluster.testacc: Error waiting for cluster (56233d4480cf49d8802ab725dd23b99f) to become ready: timeout while waiting for state to become 'normal' (last state: 'provisioning', timeout: 1h30m0s)

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.

### Expected Behavior
Should create cluster 

### Actual Behavior
Giving tiemout error

**Comment by M-Rushabh** [comment]
@Praveengostu  Please have a look.

**Comment by Praveengostu** [comment]
Hi @M-Rushabh We had fixed this issue and the fix is available in public..

**Comment by jmgalvez** [comment]
Same issue here, but a newer terraform core version. It is always failing, not randomly.

Terraform v0.11.11
IBM Cloud Provider version 0.14.1

 * ibm_container_cluster.k8s: Error waiting for initializing ingress hostname and secret: timeout while waiting for state to become 'normal' (last state: 'provisioning', timeout: 1h30m0s)

It we apply again the cluster gets created finally.

**Comment by Praveengostu** [comment]
Hi @jmgalvez The issue with "Error waiting for initializing ingress hostname and secret" is fixed and code available in public. The fix will be available from next release.

**Comment by Praveengostu** [comment]
Hi the fixes are available in release v0.15.1 and later versions. Closing this issue for now and let us know if you are able to recreate this issue
